benchmark/benchmark_slog_test.go

```diff
@@ -32,42 +32,42 @@ import (
 //
 
 func BenchmarkSlogSinkLogInfoOneArg(b *testing.B) {
-	var log logr.Logger = logr.FromSlogHandler(logr.ToSlogHandler(logr.Discard()))
+	var log logr.Logger = logr.FromSlogHandler(logr.ToSlogHandler(logr.Discard())) //nolint:staticcheck
```

Comment:
Any reason to keep the `var log logr.Logger =` instead of replacing with `log :=`?

I don't remember why I wrote it like this other than more clearly showing the type.

Reply:
I like the clarity in the benchmark.

Reply:
Probably that was why I did it - if only I remembered... :sweat_smile: